### PR TITLE
Add docs for isbare/isattached

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Review by ararslan — APPROVED** [review]
Imperative would be good for consistency otherwise lgtm

**Comment by kshyatt** [comment]
`make check-whitespace` passes locally and I fixed a few other verbs

**Review by ararslan — APPROVED** [review]
Boom, double approval

File: base/libgit2/repository.jl [hunk 1]
```diff
@@ -80,10 +80,26 @@ function headname(repo::GitRepo)
...
+"""
 function isbare(repo::GitRepo)
     return ccall((:git_repository_is_bare, :libgit2), Cint, (Ptr{Void},), repo.ptr) == 1
 end
 
+"""
+    isattached(repo::GitRepo) -> Bool
+
+Determine if `repo` is detached - that is, whether its HEAD points to a commit
+(detached) or whether HEAD points to a branch tip (attached).
```
Comment (Sacha0) [inline base/libgit2/repository.jl]: For consistency, perhaps "whether its HEAD points to ... or whether its HEAD points to ..." or simplify to "whether its HEAD points to ... or to ..."?

**Review by Sacha0 — APPROVED** [review]
lgtm! :)

File: base/libgit2/repository.jl (hunk 1 above)
Comment (tkelman) [inline base/libgit2/repository.jl]: this would be better, yes - was not addressed

**Pull request merged**